.github/workflows/iroha2-dev-pr.yml

```
@@ -102,14 +102,16 @@ jobs:
         run: |
           mv ${{ env.WASM_TARGET_DIR }}/libs ${{ env.DEFAULTS_DIR }}/libs
           mv ${{ env.DEFAULTS_DIR }}/libs/default_executor.wasm ${{ env.DEFAULTS_DIR }}/executor.wasm
-      - name: Install irohad
-        run: which irohad || cargo install --path crates/irohad --locked
+      - name: Install irohad with no default features
+        run: cargo install --path crates/irohad --locked --no-default-features
```

Add only the required feature following https://github.com/hyperledger-iroha/iroha/pull/5354#discussion_r2007110577 @s8sato:

```suggestion
      - name: Install irohad with additional features
        run: cargo install --path crates/irohad --locked -F prediction
```